lib/IRGen/GenType.cpp

```diff
@@ -1919,6 +1920,9 @@ namespace {
         return true;
 
       for (auto field : decl->getStoredProperties()) {
+        if (!field->hasInterfaceType())
```

Comment:
Thanks for fixing this crash! If these are gonna be obviated once `getInterfaceType()` is requestified, can we leave a FIXME or something on these calls so we remove them later?

Reply:
Well, resolveDeclSignature() is going away, so we'll just grep for those and delete 'em all.